docs/source/format/driver_manifests.rst

```diff
@@ -239,6 +241,12 @@ a string (single path) or a table of platform-specific paths.  The ``Driver.shar
 needed to successfully load a driver manifest.  The other keys are optional, but provide useful metadata
 about the driver.
 
+The ``manifest_version`` key is currently not required, but if present, it must be set to 1.  Driver
+manager implementations must error upon reading a manifest with ``manifest_version`` higher than 1.
+This is to prevent undefined behavior if breaking changes are made to the manifest format in the future.
+In the future, manifests that use a manifest version higher than 1 will be required to include the
+``manifest_version`` key.
+
```

Comment:
@lidavidm let me know if this and the addition in `versioning.rst` below both look OK

Reply:
Here and below I think the wording can be simplified to "manifest_version, if present, must be an integer. It defaults to 1 and can currently only be set to 1."

Reply:
I shortened and simplified the explanation in `driver_manifests.rst` but made the explanation in `versioning.rst` slightly longer and more pedantic.